src/CodeGenHelpers/ClassBuilder.cs

```diff
@@ -26,6 +26,8 @@ internal ClassBuilder(string className, CodeBuilder codeBuilder, bool partial =
             _isPartial = partial;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the ClassBuilder.